source/api_documentation/transactions_api/_advertiser_param_table.rst

```diff
@@ -173,6 +173,14 @@
     - Call Start Time (XML formatted)
     - Start of the call in Soap XML formatted time.
 
+  * - start_time_network_timezone
+    - Call Start Time Network Timezone
+    - Start of the call in the networks's time zone, followed by offset from GMT.
```

Comment:
Looks like the [spec](https://tools.ietf.org/html/rfc3339#section-5.6) allows a space.

> NOTE: ISO 8601 defines date and time separated by "T".
  Applications using this syntax may choose, for the sake of
  readability, to specify a full-date and full-time separated by
  (say) a space character.

But it seems discouraged?

> By mutual agreement the parties in information interchange may transfer the date and time format
representations. Only the date and time format representations permitted by this International Standard shall
be used. - [2004 spec](http://dotat.at/tmp/ISO_8601-2004_E.pdf)

source: [SO:9531524](http://stackoverflow.com/questions/9531524/in-an-iso-8601-date-is-the-t-character-mandatory)

Also, I don't know why I am still a watcher on this project. Ima change that. :)